plugins/joblib_launcher/hydra_plugins/joblib_launcher/conf/hydra/launcher/joblib.yaml

```diff
@@ -0,0 +1,16 @@
+hydra:
+  launcher:
+    class: hydra_plugins.joblib_launcher.JoblibLauncher
+    params:
+      joblib:
```

Comment:
did you understand my suggestion about having a hydra.joblib node?

Reply:
Yes, added it now